### PR TITLE
init

### PR DESCRIPTION
**Pull request by kostnerek**
@itspatys @kapix946 please make code review

### CODE REVIEW

File: src/containers/Confirm.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+					return "wt";
+				case 3:
+					return "śr";
+				case 4:
+					return "czw";
+				case 5:
+					return "pt";
+				case 6:
+					return "sob";
+				default:
```
Comment (itspatys) [inline src/containers/Confirm.js:30]: nice code here

Comment (kostnerek) [reply src/containers/Confirm.js:30]: thank you <3

File: src/assets/style/Confirm.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+.container > div {
+	text-align: center;
+	font-size: 2em;
+}
+.container > div:first-child {
+	margin-top: 1em;
+}
+.smol-text {
+	font-size: 1em !important;
```
Comment (itspatys) [inline src/assets/style/Confirm.css:17]: You should never use "!important"!!!😡

But I see why it is acceptable in this project.

**Review by itspatys — APPROVED** [review]
It's ok!

**Pull request merged**